jdisc_core/pom.xml

```diff
@@ -167,6 +167,11 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>vespalog</artifactId>
```

Comment:
I would like to verify the effects this dependency has on e.g. the package exports from jdisc_core (classpath) and the set of dependencies made visible via container-dev, before this PR is merged.

Reply:
OK, I will hold until green light.